### PR TITLE
[frigate] Deprecate chart

### LINKED ISSUE #92: [frigate] - relocate frigate to new home
The author of Frigate, @blakeblackshear is now hosting a [charts repo](https://github.com/blakeblackshear/blakeshome-charts) for his charts, and we should 'move' frigate to the new home.  This issue is to handle that process.

See also https://github.com/blakeblackshear/frigate/issues/156

### PR DESCRIPTION
**Pull request by bjw-s**
**Description of the change**

Since the chart is now hosted by the app author, this chart can be deprecated.

**Benefits**

**Possible drawbacks**

**Applicable issues**

- fixes #92 

**Additional information**

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Title of the PR starts with chart name (e.g. `[home-assistant]`)
- [ ] Chart is using our [common](https://github.com/k8s-at-home/charts/tree/master/charts/common) library as a dependency.
- [ ] (optional) Variables are documented in the README.md

### CODE REVIEW

File: charts/frigate/README.md [hunk 1]
```diff
@@ -1,99 +1,3 @@
...
-```
-
-Alternatively, a YAML file that specifies the values for the above parameters can be provided while installing the chart. For example,
-
-```console
-helm install --name my-release -f values.yaml stable/frigate
-```
-
-Read through the [values.yaml](https://github.com/k8s-at-home/charts/blob/master/charts/frigate/values.yaml) file. It has several commented out suggested values.
+This chart has been deprecated. Please use the official chart hosted at https://blakeblackshear.github.io/blakeshome-charts/.
```
Comment (billimek) [inline charts/frigate/README.md:3]: Suggest changing this line to:

```
This chart has been deprecated. Please use the official chart hosted at https://github.com/blakeblackshear/blakeshome-charts/tree/master/charts/frigate.
```

File: charts/frigate/templates/NOTES.txt [hunk 2]
```diff
@@ -1,19 +1 @@
...
-     NOTE: It may take a few minutes for the LoadBalancer IP to be available.
-           You can watch the status of by running 'kubectl get svc -w {{ include "frigate.fullname" . }}'
-  export SERVICE_IP=$(kubectl get svc --namespace {{ .Release.Namespace }} {{ include "frigate.fullname" . }} -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
-  echo http://$SERVICE_IP:{{ .Values.service.port }}
-{{- else if contains "ClusterIP" .Values.service.type }}
-  export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l "app.kubernetes.io/name={{ include "frigate.name" . }},app.kubernetes.io/instance={{ .Release.Name }}" -o jsonpath="{.items[0].metadata.name}")
-  echo "Visit http://127.0.0.1:5000 to use your application"
-  kubectl port-forward $POD_NAME 5000:5000
-{{- end }}
+This chart has been deprecated. Please use the official chart hosted at https://blakeblackshear.github.io/blakeshome-charts/.
```
Comment (billimek) [inline charts/frigate/templates/NOTES.txt:1]: Suggest changing this line to:

```
This chart has been deprecated. Please use the official chart hosted at https://github.com/blakeblackshear/blakeshome-charts/tree/master/charts/frigate.
```

**Review by billimek — CHANGES_REQUESTED** [review]
Can we use the new chart GitHub repo URL instead of the helm repository URL?  There won't be much discoverable by going to https://blakeblackshear.github.io/blakeshome-charts/

**Comment by bjw-s** [comment]
Sure thing, will modify them in a bit

**Comment by bjw-s** [comment]
Done

**Review by billimek — APPROVED** [review]
LGTM, thanks for making these changes @bjw-s!

**Pull request merged**